src/types.rs

```diff
@@ -100,6 +100,17 @@ pub struct Pagination {
     pub pages: u32,
 }
 
+#[derive(Clone,Debug,RustcDecodable)]
```

Comment:
could you please add a doc comment here explaining what this type is since there doesn't seem to  be a slack page for the type?
Something like https://github.com/neil-lobracco/slack-rs-api/blob/ff69d2c6e584e857e2ab81ea763252e15eee81ee/src/types.rs#L94 should be fine.